### ISSUE TITLE
Run redis-server from a different Docker container 

### ISSUE THREAD

**Issue by samgiles** [fxbox/registration_server#26]
Instead of running the server and redis in the same container, we should link them and run only the server in the container this project's docker file describes.

**Comment by michielbdejong** [comment]
Good point - same is probably true for https://github.com/fxbox/dns-server/blob/master/server/run.sh which combines pagekite + dns-server in one Docker container. When you do, remember to update the wiki at https://wiki.mozilla.org/Connected_Devices/Projects/Project_Link/Registration_Server_and_Tunneling.
